app/assets/javascripts/surveys.js

```diff
@@ -720,6 +753,14 @@
       return generateCookieName('survey_seen_' + survey.identifier)
     },
 
+    isBeingViewedOnMobile: function () {
+      return window.matchMedia( "(max-width: 800px)" ).matches
```

Comment:
Same comment as above, but if not, can we somehow make this conditional, and the one in `tlsSurveyUrl` match up?  A global constant for the query or something?